tests/resources/aria-at-test-run.mjs

```diff
@@ -185,10 +201,24 @@ export function instructionDocument(resultState, hooks) {
         commands: {
           description: `${lastInstruction} ${commandListInstructions}`,
           commands: commands.map((command, index) => {
-            const { description: settings, text: screenText } = commandSettings[index];
-            return `${command}${
-              screenText && settings !== 'defaultMode' ? ` (${screenText})` : ''
-            }`;
+            const {
+              description: settings,
+              text: screenText,
+              additionalSettingsExpanded = [],
+            } = commandSettings[index];
+
+            const hasScreenText = screenText && settings !== 'defaultMode';
+            const hasadditionalSettings = hasScreenText && additionalSettingsExpanded.length > 0;
```

Comment:
I go back and forth on this. I get the logic of keeping the camel case from the `additionalSettings` but having a variable without normal capitalization does feel odd